.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
```

Comment:
I'm not sure what this file is supposed to be for?

Reply:
This is how you manage the pre-commit hook framework called `pre-commit`. https://pre-commit.com/ I included it because it declaratively shows how the formatting of the codebase has been arrived at and allows for trivial changes. E.g. Whenever CMS migrated to Python 3, just 

```diff
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index cd24669..e902454 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -3,4 +3,4 @@ repos:
     rev: v2.19.4
     hooks:
     - id: pyupgrade
-      args: ["--py3-plus"]
+      args: ["--py37-plus"]
```
then run

```console
$ pre-commit run --all-files
```

commit the changes and you're done with no time wasted.

Reply:
Didn't know that tool... Looks useful, thanks!